Code/GraphMol/PeriodicTable.cpp

```diff
@@ -79,9 +83,18 @@ PeriodicTable::PeriodicTable() {
   }
 }
 
+#if RDK_BUILD_THREADSAFE_SSS
+boost::mutex ptMutex;
+#endif
+
 PeriodicTable *PeriodicTable::getTable() {
   if (ds_instance == 0) {
-    ds_instance = new PeriodicTable();
+#if RDK_BUILD_THREADSAFE_SSS
+    boost::lock_guard<boost::mutex> lock(ptMutex);
```

Comment:
There's actually no reason for the returned result to not be const: the class has no public data members and no non-const methods. However, switching to a const pointer will break a lot of code.